config/core/deployments/sources-controller.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO(n3wscott): DELETE THIS FILE AFTER v0.13 CUTS
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sources-controller
+  namespace: knative-eventing
+  labels:
+    eventing.knative.dev/release: devel
```

Comment:
Does it make sense to add an annotation or label signaling that this deployment is deprecated and intentionally scaled to zero? I'm thinking if someone with a malfunctioning source is trying to debug, they might see this and try to scale it up manually.

Reply:
I don't know what we have done before. for a v1alpha1 api this alone is nice thing to do...

Reply:
Fair point :smile: